source/_integrations/vitrea.markdown

```diff
@@ -0,0 +1,110 @@
+---
+title: Vitrea
+description: Instructions on how to integrate Vitrea home automation devices with Home Assistant.
+ha_category:
+  - Cover
+  - Switch
+  - Number
+ha_release: 2025.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - "@solangek"
+ha_domain: vitrea
+ha_platforms:
+  - cover
+  - number
+  - switch
+ha_integration_type: hub
+---
+
+The Vitrea integration allows you to control Vitrea home automation devices through Home Assistant. This integration supports switches, covers (blinds), and timer (boiler) controls for your Vitrea system.
+
+## Prerequisites
+
+- A Vitrea hub connected to your local network
+- The Vitrea hub must be accessible via TCP/IP
+- Default connection uses port 11502
+
+## Configuration
+
+The Vitrea integration can be configured through the Home Assistant user interface.
+
+### Setup via the frontend
+
+1. Go to **Settings** → **Devices & Services**
+2. Click **Add Integration**
+3. Search for "Vitrea" and select it
+4. Enter your Vitrea hub's IP address and port (default: 11502)
+5. Click **Submit**
```

Comment:
```suggestion
{% include integrations/config_flow.md %}
```
There's a [standard text block](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration), see also the [example page](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration),